content/ko/docs/concepts/scheduling-eviction/pod-priority-preemption.md

```diff
@@ -203,7 +203,7 @@ spec:
 정보를 제공한다.
 
 파드 P는 반드시 "지정된 노드"로 스케줄링되지는 않는다.
-The scheduler always tries the "nominated Node" before iterating over any other nodes.
+스케줄러는 다른 노드를 반복하기 전에 항상 "nominated Node"를 시도합니다.
 스케줄러는 다른 노드에 스케줄링을 시도하기 전에 항상 "지정된 노드"부터 시도한다.
```

Comment:
Hi @Pradumnasaraf 
Thank you for the contribution!

As a native Korean, the translated sentence is not natural to me. (is it machine translation?) 
https://github.com/kubernetes/website/issues/37740#issuecomment-1304755015
and, seem not compatible with the Korean localization style https://kubernetes.io/ko/docs/contribute/localization_ko/ 

But, I was realized the untranslated English sentence was actually translated in the next line.
So, we can just remove the 206 line (the English sentence)
```suggestion
스케줄러는 다른 노드에 스케줄링을 시도하기 전에 항상 "지정된 노드"부터 시도한다.
```

Korean localization uses own development branch. 
Please change base of this PR to [dev-1.25-ko.1](https://github.com/kubernetes/website/tree/dev-1.25-ko.1) which is the current development branch for Korean localization.

Reply:
Hey,
Yes, I take some help from machine translation, as I am learning Korean. Thank you for the review

Reply:
@Pradumnasaraf 
Good to know you are learning Korean :)
Thanks for the contribution~!